tests/console/detect_yast2_failures.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
```

Comment:
please, extend copyright date in all files.

Reply:
Done